ipushp/__manifest__.py

```diff
@@ -21,6 +21,8 @@
         'views/find_contacts.xml',
     ],
     'images': ['static/description/page_1.png'],
+    'price': 0,
```

Comment:
Are you really serious?
nowhere this is touching https://github.com/JayVora-SerpentCS/SerpentCS_Contributions/blob/10.0/ipushp/__manifest__.py at all.
name itself is different.
@Chandresh-SerpentCS